dev/TreeView/TreeView.cpp

```diff
@@ -264,7 +264,7 @@ void TreeView::OnPropertyChanged(const winrt::DependencyPropertyChangedEventArgs
     {
 
         const auto items = SelectedItems();
-        const auto selected = items.Size() > 0 ? items.GetAt(0) : nullptr;
+        const auto selected = (items != nullptr && items.Size() > 0) ? items.GetAt(0) : nullptr;
```

Comment:
>items != nullptr [](start = 31, length = 16)

Under what circumstances is this null?

Reply:
It seems that SelectedItems returns null when TreeView can't resolve it's listcontrol.

Reply:
and what causes that to happen?

Reply:
SelectedItems returns null if the internal listview can't be found or the listviews ListViewModel can't be found. So in #3129, we probably crashed due to the binding resolving before the template was applied leaving us in a state where we did not have a listview or selectionmodel.